src/Bicep.Core.IntegrationTests/ParameterFileTests.cs

```diff
@@ -94,9 +94,37 @@ public void Parameters_file_cannot_reference_non_existing_env_variable()
 
         result.ExcludingLinterDiagnostics().Should().HaveDiagnostics(new[]{
                 ("BCP338", DiagnosticLevel.Error,
-                "Failed to evaluate parameter \"stringEnvVariable\": Environment variable does not exist, and no default value set")});
```

Comment:
Isn't the error message wrong?  It says the parameter was 'stringEnvVariable', but that's the name of the environment variable.  Should it be:

Failed to evaluate parameter \"fromEnv\": Environment variable \"stringEnvVariable\" does not exist, and no default value set"

?

Reply:
You are right. Since the wording wasn't part of my change I didn't pay much attention to it. I can fix that as well while I'm at it.